.pylintrc

```diff
@@ -0,0 +1,563 @@
+[MASTER]
```

Comment:
Should this file live in `/backend` instead of in the project root?

Reply:
I tried to put it under /backend, but it seems that VS Code is looking in the project root for this config file.

However, with ESLint, I found a way to specify where the config file is. For ESLint, I'll put the config file under /frontend.